ReactWindows/ReactNative.Shared/UIManager/LayoutShadowNode.cs

```diff
@@ -10,7 +10,11 @@ namespace ReactNative.UIManager
     /// </summary>
     public class LayoutShadowNode : ReactShadowNode
     {
+#if WINDOWS_UWP
         private const float Undefined = CSSConstants.Undefined;
+#else
+        private const float Undefined = 0.0f;
```

Comment:
Fair enough, and on a hunch I checked and VS is no longer complaining about this either.